user/database-setup.md

```diff
@@ -170,34 +170,33 @@ before_script:
 
 ### Using a different PostgreSQL Version
 
-The Travis CI build environments use version 9.1 by default, but other versions from the official [PostgreSQL APT repository](http://apt.postgresql.org) are also available. To use a version other than the default, specify only the **major.minor** version in your `.travis.yml`:
+The Travis CI build environments use version 9.1 by default, but other versions
+from the official [PostgreSQL APT repository](http://apt.postgresql.org) are
+also available. To use a version other than the default, specify only the
+**major.minor** version in your `.travis.yml`:
 
 ```yaml
 addons:
   postgresql: "9.4"
 ```
 {: data-file=".travis.yml"}
 
-The following versions are available on Linux builds:
+Many PostgreSQL versions have been preinstalled in our build environments, and
+others may be added and activated at build time by using a combination of the
+`postgresql` and `apt` addons along with a global env var override for `PGPORT`:
 
-| PostgreSQL | sudo enabled precise | sudo enabled trusty | container precise | container trusty |
-|:----------:|:--------------------:|:-------------------:|:-----------------:|:----------------:|
-|    9.1     |         yes          |                     |        yes        |                  |
-|    9.2     |         yes          |         yes         |        yes        |       yes        |
-|    9.3     |         yes          |         yes         |        yes        |       yes        |
-|    9.4     |         yes          |         yes         |        yes        |       yes        |
-|    9.5     |         yes          |         yes         |                   |       yes        |
-|    9.6     |                      |         yes         |                   |       yes        |
-
-On OS X, the following versions are installed:
-
-|     image     | version |
-|:-------------:|:-------:|
-|    xcode61    |   9.3   |
-| beta-xcode6.1 |   9.3   |
-|   xcode6.4    |   9.4   |
-|   xcode7.3    |   9.5   |
-|    xcode8     |   9.5   |
+``` yaml
+addons:
+  postgresql: "10"
+  apt:
+    packages:
+    - postgresql-10
+    - postgresql-client-10
+env:
+  global:
+  - PGPORT=5433
+```
+{: data-file=".travis.yml"}
```

Comment:
Hey @meatballhat - thanks for the PR. :100: on losing the nasty-and-soon-to-be-outdated table.

Do you think it is worth mentioning either of the following:

* you need *both* postrgrsql and the apt packages and they must match (I know I know)
* what PGPORT is :-p (I'm also assuming if you use Postgre you'll know this, but still) (Also, when you say override, is there a default? )